clarifai/cli/deployment.py

```diff
@@ -0,0 +1,49 @@
+import click
+from clarifai.cli.base import cli
+from clarifai.client.nodepool import Nodepool
+from clarifai.utils.cli import display_co_resources
+
+
+@cli.group()
+@click.option(
+    '--nodepool_id', required=True, help='Nodepool ID for the Nodepool to interact with.')
+@click.pass_context
+def deployment(ctx, nodepool_id):
+  """Manage Deployments: create, delete, list"""
+  ctx.obj = Nodepool(
+      nodepool_id=nodepool_id,
+      user_id=ctx.obj['user_id'],
+      pat=ctx.obj['pat'],
+      base_url=ctx.obj['base_url'])
+
+
+@deployment.command()
+@click.option('--deployment_id', required=True, help='Deployment ID for the deployment to create.')
```

Comment:
We can replace whatever relevant alias we fix for `deployment` in here

Reply:
Done as `dpl`